expo/scheduler/functions.py

```diff
@@ -436,3 +437,121 @@ def get_roles_from_scheduler(workeritems, conference):
             roles += [allocation.resource.as_subtype.role]
 
     return list(set(roles))
+
+
+def calendar_export(conference=None,
+                    cal_format='gbook',
+                    event_types='All',
+                    day=None,
+                    ):
+    '''
+    View to export calendars, formatted in either iCalendar format, or as
+    a csv file.  Used to allow importing of the calendar information into
+    other applications, such as Guidebook, or Calendar applications.
+
+    conference  - Slug for the conference we want to export data for
+    cal_format  - Which format to use for the exported data, can be 'gbook'
+                  for Guidebook csv, 'ical' for iCal calendar programs (ics)
+    event_types - Which event types to include, usually will be 'All' to
+                  generate a complete calendar, but could also be 'Class',
+                  'Show', etc, or a list or tuple of event_types
+    day         - Which day of the conference to get calendar info for,
+                  None or 'All' gets info for entire conference
+    '''
+    #  TODO: Add ability to filter on a users schedule for things like
+    #  volunteer shifts.
+
+    from gbe.functions import get_conference_by_slug
+    import gbe.models as conf
```

Comment:
Please  pull only the models you need here.
And do you have a circularity problem?  If not, I'd advise pulling at the start of the file.